app/controllers/ontologies_controller.rb

```diff
@@ -13,7 +13,7 @@ class OntologiesController < ApplicationController
   include SparqlHelper
   include SubmissionFilter
   include OntologyContentSerializer
-  include UriRedirection
+  include OntologiesRedirectionController
```

Comment:
no need to include it here, just make it a class that extends ApplicationController, so it can be a controller like the others, and update the route to point to him.